appengine_flexible/go115_and_earlier/go.mod

```diff
@@ -0,0 +1,37 @@
+module github.com/GoogleCloudPlatform/golang-samples/appengine_flexible
```

Comment:
I'm not sure if it would work to keep this module path the same if the copy is still present at the original path.

Reply:
That's fair. I've updated the module path now. In my brief testing, it seemed like I always ended up deleting the `go.mod` files in this and the parent directory, to get `gcloud app deploy` to work (gopath mode).